var/spack/repos/builtin/packages/py-mosek/package.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import platform
+import sys
+
+from spack.package import *
+
+
+class PyMosek(PythonPackage):
+    """MOSEK is a package for large-scale convex and mixed-integer optimization, including LP, QP, SOCP, exponential and power cone problems, SDP, QCQP and MIP. Documentation and examples can be found at the https://www.mosek.com/documentation. MOSEK requires a license. Free of charge trial licenses as well as licenses for non-commercial academic use can be obtained from https://www.mosek.com."""
+
+    homepage = "https://www.mosek.com"
+
+    url = "https://pypi.org/project/Mosek/"
+
+    maintainers("bourdin")
+
+    license("https://mosek.com/products/license-agreement", checked_by="bourdin")
+
+    arch = platform.machine()
+    os = sys.platform
```

Comment:
No need for `import sys` if `import platform` already.
```suggestion
    system = platform.system().lower()
```